moveit_ros/moveit_servo/src/pose_tracking.cpp

```diff
@@ -307,8 +307,23 @@ geometry_msgs::TwistStampedConstPtr PoseTracking::calculateTwistCommand()
   return msg;
 }
 
+void PoseTracking::stopMotion()
+{
+  stop_requested_ = true;
+
+  // Send a 0 command to Servo to halt arm motion
+  auto msg = moveit::util::make_shared_from_pool<geometry_msgs::TwistStamped>();
```

Comment:
Is this really enough? It seems like this could go in after `stop_requested_` so that it has no effect.

Reply:
It looks fine to me. `stop_requested_` stops the Pose Tracking loop. Publishing the zero twist stops the Servo loop.